<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PraxisDlg</class>
 <widget class="QDialog" name="PraxisDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="_pBottomLab">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>630</y>
     <width>1198</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1198</width>
     <height>33</height>
    </size>
   </property>
   <property name="text">
    <string>所有内容由武汉巴赫风文化传媒倾心打造，版权归巴赫风所有</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="_pMenuBtnParentWidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>594</y>
     <width>948</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>948</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="_pSbumitBtn">
   <property name="geometry">
    <rect>
     <x>1075</x>
     <y>591</y>
     <width>85</width>
     <height>48</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>48</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="_pCloseBtn">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>10</y>
     <width>40</width>
     <height>43</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>43</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
